api/src/main/java/module-info.java

```diff
@@ -16,29 +16,8 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import jakarta.data.Limit;
-import jakarta.data.Order;
-import jakarta.data.Sort;
-import jakarta.data.metamodel.StaticMetamodel;
-import jakarta.data.page.PageRequest;
-import jakarta.data.repository.BasicRepository;
-import jakarta.data.repository.By;
-import jakarta.data.repository.CrudRepository;
-import jakarta.data.repository.DataRepository;
-import jakarta.data.repository.Delete;
-import jakarta.data.repository.Find;
-import jakarta.data.repository.Insert;
-import jakarta.data.repository.OrderBy;
-import jakarta.data.repository.Param;
-import jakarta.data.repository.Query;
-import jakarta.data.repository.Repository;
-import jakarta.data.repository.Save;
-import jakarta.data.repository.Select;
-import jakarta.data.repository.Update;
-
-import java.util.concurrent.CompletableFuture;
-import java.util.concurrent.CompletionStage;
 import java.util.Set;
+import java.util.concurrent.CompletableFuture;
```

Comment:
This comment will help to prevent future attempts at removal:
```suggestion
// Although these appear to be unused, these imports are needed by Javadoc.
import jakarta.data.Limit;
import jakarta.data.Order;
import jakarta.data.Sort;
import jakarta.data.metamodel.StaticMetamodel;
import jakarta.data.page.PageRequest;
import jakarta.data.repository.BasicRepository;
import jakarta.data.repository.By;
import jakarta.data.repository.CrudRepository;
import jakarta.data.repository.DataRepository;
import jakarta.data.repository.Delete;
import jakarta.data.repository.Find;
import jakarta.data.repository.Insert;
import jakarta.data.repository.OrderBy;
import jakarta.data.repository.Param;
import jakarta.data.repository.Query;
import jakarta.data.repository.Repository;
import jakarta.data.repository.Save;
import jakarta.data.repository.Select;
import jakarta.data.repository.Update;

import java.util.concurrent.CompletableFuture;
import java.util.concurrent.CompletionStage;
import java.util.Set;
```